src/components/CalculatorDisplay.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function CalculatorDisplay() {
+  return (
+    <div className="calc">
+      <span className="keys4">0</span>
+    </div>
+  );
+}
+
```

Comment:
- [ ] [OPTIONAL] Nice job on implementing React router correctly, however, when a values are entered for calculation, the results are not cleared when a new calculation is being performed. For instance, if the results of my previous calculation was `205`, and then when I press the `9` button in an attempt to make a new calculation it displays `2059` on the screen. Let's ensure that the screen is cleared on performing a new calculation.